book/01.md

```diff
@@ -238,29 +238,29 @@ End of TN.
 
 The DB instruction is an abbreviation for "Data Byte". That is to write 1 byte directly to the file. I like writing assembly instructions in uppercase, but they can be written in lowercase like "`db`" without any problems.
 
-This instruction is the strongest ultimate weapon in the world of assembler, that is, any data (and any program, by the way) can be written with just that instruction. That's why there are no files that can't be created in assembler. You can create text files, image files, whatever you're told to do, you can do it in assembler. This versatility(?) is rare in other languages (e.g. C).
+This instruction is the ultimate strongest weapon in the world of assembler. That is, any data (and any program, by the way) can be written with just that instruction. That's why there are no files that can't be created in assembler. You can create text files, image files, whatever you're told to do, you can do it in assembler. This expressiveness is rare in other languages (e.g. C).
 
 The RESB instruction is an abbreviation of "Reserve Byte", with a meaning of something like "For now just leave 10 bytes there" you'd use it like RESB 10. And these 10 bytes are reserved (imagine something like a reserved seat). But in NASK it doesn't just reserve the bytes but to fill them with zeroes. So we're using this command to write a bunch of 0x00. Thanks to this instruction I don't have to write 180000 lines, which is a big help.
 
 ---
 
-TN: The implication is that in NASM it will reserve zeroes but, will hesitate to fill them with zeroes. While actually filling the file with zeroes it prints warning like this:
+TN: The implication is that in NASM it will also reserve zeroes but, it will hesitate. While actually filling the file with zeroes it prints a warning like this:
```

Comment:
Should be "reserve space", not "reserve zeroes", that was a thoughtless typo on my side. Reserving zeroes makes no sense to begin with :D

Reply:
You're right, that does make more sense. Changed it to reserve space. Should be resolved now!